src/a5/a5_file.c

```diff
@@ -66,7 +66,7 @@ static char *ff_get_filename(const char * path)
 {
     char * p = (char *)path + strlen(path);
 
-    while((p > path) && (*(p - 1) != '/') && (*(p - 1) != OTHER_PATH_SEPARATOR))
+    while((p > path) && (*(p - 1) != OTHER_PATH_SEPARATOR))
     {
```

Comment:
I didn't have the very latest changes before pushing this PR, and now I see there was already a commit to fix this bit. keeping them in this PR because I don't think you need to check both slashes

Reply:
It's probably wise to check for both slashes for better compatibility. Windows users may still be using '/' as Windows supports '/' as well as '\\' for path separators. If we only check for `OTHER_PATH_SEPARATOR`, users who just used '/' for everything will run into issues when moving from Allegro 4 to Allegro Legacy if they rely on these specific routines.